common/src/test/java/com/blackship/battlesheep/common/communication/network/packet/PacketFactoryTest.java

```diff
@@ -0,0 +1,25 @@
+package com.blackship.battlesheep.common.communication.network.packet;
+
+import com.blackship.battlesheep.common.communication.packet.Packet;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+/**
+ * @author milosz
+ * @since 7/27/17
+ */
+public class PacketFactoryTest {
+
+    @Test
+    public void shouldCreateNetworkMovePacket() {
+
+        //given
+
+        //when
+        Packet givenPacket = PacketFactory.createMove();
+
+        //then
+        assertTrue(givenPacket instanceof NetworkPacketMove);
```

Comment:
it's always true according to intellij, and it was classified as "Probable bug" :exclamation: